.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 npm-debug.log
+.idea
```

Comment:
You should add this to your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) since it's specific to your workstation :)